React/CoreModules/RCTDevSettings.mm

```diff
@@ -405,17 +405,18 @@ - (void)addHandler:(id<RCTPackagerClientMethod>)handler forPackagerMethod:(NSStr
 
 - (void)setupHMRClientWithBundleURL:(NSURL *)bundleURL
 {
-  if (bundleURL && !bundleURL.fileURL) { // isHotLoadingAvailable check
```

Comment:
I assume this comment was a _TODO_ that's now done :trollface: 
:wave: @PeteTheHeat